HoloLensCommander/HoloLensCommander/Utilities/AppResponse.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HoloLensCommander
+{
+    public class AppResponse
```

Comment:
Based on this class definition, we can easily eliminate the newtonsoft dependency.

Reply:
@davidkline-ms Sounds good; How do you recommend deserializing without newtonsoft?

Reply:
Over in the WindowsDevicePortalWrapper project (https://github.com/Microsoft/WindowsDevicePortalWrapper), take a look at how the OperatingSystemInformation object is defined (https://github.com/Microsoft/WindowsDevicePortalWrapper/blob/master/WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/Core/OsInformation.cs).

The ResponseHelpers.cs file (https://github.com/Microsoft/WindowsDevicePortalWrapper/blob/master/WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/HttpRest/ResponseHelpers.cs) shows how we validate the data format and read the contents.

As mentioned before, i am happy to help with this as i put together the original version of the WDPW json code.